yt_dlp/extractor/youtube.py

```diff
@@ -2487,15 +2419,6 @@ def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, pl
             player_ytcfg = master_ytcfg if client == 'web' else {}
             if 'configs' not in self._configuration_arg('player_skip'):
                 player_ytcfg = self._extract_player_ytcfg(client, video_id) or player_ytcfg
-                if client == 'web_embedded':
```

Comment:
doesn't hurt keeping the client config download in, but we don't need this weird check specific for web_embedded anymore imo